assembly_image.py

```diff
@@ -0,0 +1,129 @@
+
+
+def get_wardrobe_width(wardrobe):
+    return sum([wardrobe[i]["width"] for i in range(len(wardrobe))])
```

Comment:
![imatge](https://github.com/user-attachments/assets/d52a6281-d754-4095-9ed5-6e47998e0bd4)

```suggestion
def get_wardrobe_width(wardrobes):
    return sum([wardrobe["width"] for wardrobe in range(len(wardrobes))])
```